### ISSUE TITLE
Feature Request: Allow meta data for portal creation items

### ISSUE THREAD

**Issue by Darkosto** [TeamTwilight/twilightforest#167]
# Name of item used to create the Twilight Forest Portal
    S:portalCreationItem=astralsorcery:itemcraftingcomponent:1

Would you please consider adding the ability to use an item with meta data in the above config entry?

MC: 1.12.1
TW: 2.4.0.58

Thank you!
Darkosto

**Comment by Drullkus** [comment]
Fixed via d9f6fdd

**Comment by AtomicBlom** [comment]
@Darkosto Just a heads up that I made a change to support "ignore metadata". 
```
S:portalCreationItem=minecraft:red_flower
I:portalCreationMeta=5
```

Will only create a portal for orange tulips.

```
S:portalCreationItem=minecraft:red_flower
I:portalCreationMeta=-1
```
Will allow any of the red_flower variants to create the portal.
